src/api/crosscutRequests.js

```diff
@@ -0,0 +1,73 @@
+import ky from 'ky'
+import { fetchGeoJSON, fetchCurrentAttributes } from './requests';
+import { getToken } from '../services/JWTManager'
+const CROSSCUT_API = "https://qwui27io74.execute-api.us-east-1.amazonaws.com";
```

Comment:
We should get this behind a crosscut created url, rather than this AWS created one so that we can handle problems like this domain being accidentally deleted.